src/Ionide.ProjInfo.ProjectSystem/FSIRefs.fs

```diff
@@ -23,55 +24,15 @@ type TFM =
     | NetFx
     | NetCore
 
-[<NoComparison>]
-type NugetVersion =
```

Comment:
ripped this out in favor of SemanticVersioning library to share the implementation